log/wrapper.go

```diff
@@ -60,3 +61,22 @@ func ZapWrapper(level Level) Wrapper {
 		f(msg)
 	}
 }
+
+// ErrorWithRavenWrapper is a Wrapper implementation that both use Zap logger to
+// log at error level, and send the error to raven.
+//
+// In most cases this should be the one used to pass into Baseplate.go libraries
+// expecting a log.Wrapper.
+//
+// Note that this should not be used as the logger set into thrift.TSimpleServer,
+// as that would use the logger to log network I/O errors,
+// which would be too spammy to be sent to raven.
+func ErrorWithRavenWrapper(msg string) {
```

Comment:
While we're at it, we should rename Raven to Sentry. In the Go library they also moved away from the word raven https://github.com/getsentry/sentry-go

Reply:
We are currently using `github.com/gensentry/raven-go`: https://github.com/reddit/baseplate.go/blob/e0f1201fa38d0132ba5acfab621435fef7b754dd/log/log.go#L4

I'm not against migrating to sentry-go instead, but sentry-go requires explicit initialization, while raven-go does not. That can happen in a future PR instead.

Reply:
It's still Sentry we're logging though right? I think they figured Raven vs. Sentry is just confusing.

Reply:
what I mean is it's probably better renamed when we migrated to the sentry-go library :)

Reply:
Sounds good :+1: